### ISSUE TITLE
There is no way to use string-based idColumn

### ISSUE THREAD

**Issue by itmammoth** [itmammoth/Tamotsu#9]
From #7

**Comment by itmammoth** [comment]
b6f250e

**Comment by itmammoth** [comment]
Resolved with b6f250e

**Comment by itmammoth** [comment]
You can use id as string like the following code.
```
var Food = Tamotsu.Table.define({
  sheetName: 'Food',
  idColumn: 'Name',
  autoIncrement: false,
}
var food = Food.create({ 'Name': 'Tempura', 'Price': '1480' });
```
